yarn.lock

```diff
@@ -7650,18 +6831,18 @@ semver@7.5.3:
   dependencies:
     lru-cache "^6.0.0"
 
-semver@7.x, semver@^7.0.0, semver@^7.1.1, semver@^7.2.1, semver@^7.3.2, semver@^7.3.4, semver@^7.3.5, semver@^7.3.7, semver@^7.3.8:
+semver@^6.0.0, semver@^6.3.0, semver@^6.3.1:
```

Comment:
these are due to Babel with is a jest dependency. Babel has stated that their use of semver does not allow the vulnerability to be exploited so we can ignore the security warning.